src/components/RecordDetail/PlaceMap.tsx

```diff
@@ -0,0 +1,32 @@
+import { Controls, Peripleo } from '@peripleo/peripleo';
+import { Map, Zoom } from '@peripleo/maplibre';
+import style from './mapstyle.json';
+import { LocationMarkers } from '@performant-software/geospatial';
+
+interface Props {
+  record: any;
+}
+
+const PlaceMap = (props: Props) => {
+  return (
+    <Peripleo>
+      <Map
+        style={style}
```

Comment:
Is there any reason we're not using the map styles specified in `config.json`?

Reply:
Which config object contains map styles? I see `layers`, but I don't see `version` or `sources` in the Scavenger Hunt config file.

Reply:
You should be able to use [PeripleoUtils](https://github.com/performant-software/react-components/blob/master/packages/core-data/src/utils/Peripleo.js#L58) to convert the layers defined in the config. See [MapView](https://github.com/performant-software/core-data-places/blob/develop/src/apps/search/MapView.tsx) for an example.